n9kv/docker/Dockerfile

```diff
@@ -1,5 +1,9 @@
 FROM ghcr.io/srl-labs/vrnetlab-base:0.1.0
```

Comment:
can you also change this to `ghcr.io/srl-labs/vrnetlab-base:0.2.1`

This is the most recent base image